### DISCUSSION TITLE
ImageRepository cosign verify support

### DISCUSSION THREAD

**Discussion by BobyMCbobs** [fluxcd/flux2#4682] [Proposals] [answered]
It would be very powerful to be able to verify _ImageRepository_ within the tooling of image updates to verify with cosign or GPG just like what is currently possible with OCIRepository under verify.

**Comment by makkes** [comment, marked as answer]
- https://github.com/fluxcd/image-automation-controller/issues/608
- https://github.com/fluxcd/image-automation-controller/issues/355
